src/components/Menubar.js

```diff
@@ -33,21 +33,21 @@ class Menubar extends Component {
             <Menu pointing secondary>
                 <Menu.Item
                 name='dashboard'
-                active={activeItem === 'dashboard'}
+                active={activeItem === 'dashboard' || activeItem === '/dashboard'}
                 onClick={this.handleItemClick}
                 as={Link}
                 to={dashboard()}
                 > DASHBOARD </Menu.Item>
                 <Menu.Item
                 name='forms'
-                active={activeItem === 'forms'}
+                active={activeItem === 'forms' || activeItem === '/forms'}
                 onClick={this.handleItemClick}
                 as={Link}
                 to={forms()}
                 > FORMS </Menu.Item>
                 <Menu.Item
                 name='submissions'
-                active={activeItem === 'submissions'}
+                active={activeItem === 'submissions' || activeItem === '/submissions'}
```

Comment:
```suggestion
                active={activeItem === 'submissions' || activeItem === '/submission'}
```
URL contains `submission`.